bindata/network/network-diagnostics/network-check-tester.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: network-check-tester
+  namespace: openshift-network-diagnostics
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: network-check-tester
+  template:
+    metadata:
+      labels:
+        app: network-check-tester
+    spec:
+      serviceAccountName: network-check-tester
+      containers:
+      - name: network-check-tester-container
```

Comment:
What does this do? I don't think we can reference `busybox` as an image.

Reply:
Yeah, this was just for testing. So basicly, I need a pod to act as a src to run the connectivity checks so in the PR I have this pod with a busybox container doing nothing and the check-endpoints sidecar containers that's doing the check business logic.

Reply:
which image should I use instead for this scenario?

Reply:
nm, as per our slack chat doesn't make sense to have a dummy container, just run check-endpoints on the pod.

Reply:
done